src/video-player.js

```diff
@@ -134,6 +135,28 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     this.videojs.clearTimeout(this.reTryId);
   };
 
+  _setRetryForLiveStream(delaySeconds = 10) {
+    const tempVideo = document.createElement('video');
+    tempVideo.setAttribute('crossorigin', 'anonymous');
+    const liveStreamUrl = this.currentSourceUrl();
+
+    const retryLiveStreamLoad = () => {
+      setTimeout(() => {
+        tempVideo.src = liveStreamUrl;
+        tempVideo.load();
+      }, delaySeconds * 1000);
+    };
+
+    tempVideo.onprogress = () => {
+      tempVideo.onerror = null;
+      tempVideo.oncanplaythrough = null;
+      this.source(liveStreamUrl);
+    };
+    tempVideo.onerror = retryLiveStreamLoad;
+
+    retryLiveStreamLoad();
+  }
```

Comment:
@jakub-roch we already have a reTryVideo handler.
can we re-use that instead?
https://github.com/cloudinary/cloudinary-video-player/blob/master/src/video-player.js#L418